playbooks/roles/backups/defaults/main.yml

```diff
@@ -3,15 +3,16 @@
 BACKUPS_MYSQL: yes
 BACKUPS_MONGO: yes
 BACKUPS_BUCKET: ''
-BACKUPS_AWS_ACCESS_KEY_ID: ''
-BACKUPS_AWS_SECRET_ACCESS_KEY: ''
 BACKUPS_DIR: '/tmp/db_backups'
 BACKUPS_SENTRY_DSN: ''
 
 # gs: Google Cloud Storage
 # s3: Amazon Simple Storage Service
 BACKUPS_PROVIDER: 'gs'
 
+# AWS
+BACKUPS_AWS_ACCESS_KEY_ID: ''
+BACKUPS_AWS_SECRET_ACCESS_KEY: ''
 
 # Google Cloud Storage
```

Comment:
@johnbaldwin those two are for AWS, not for Google Cloud Storage. I think are well grouped already.